packages/lodestar/src/network/discovery/discv5.ts

```diff
@@ -0,0 +1,75 @@
+import {EventEmitter} from "events";
+import randombytes from "randombytes";
+import PeerInfo from "peer-info";
+import Multiaddr from "multiaddr";
+import {
+  Discv5,
+  ENR,
+  createNodeId,
+} from "@chainsafe/discv5";
+
+export interface IDiscv5DiscoveryInputOptions {
+  enr: ENR;
+  bindAddr: Multiaddr;
+  bootEnrs: ENR[];
+}
+
+export interface IDiscv5DiscoveryOptions extends IDiscv5DiscoveryInputOptions {
+  peerInfo: PeerInfo;
+}
+
+/**
+ * Discv5Discovery
+ */
+export class Discv5Discovery extends EventEmitter {
```

Comment:
I think we should release this class as libp2p-discv5 or inside @chainsafe/discv5 along with instructions how to add to libp2p stack?

Reply:
:+1: